apps/ledger-live-desktop/src/renderer/screens/customImage/Step4Transfer.tsx

```diff
@@ -36,14 +39,31 @@ const StepTransfer: React.FC<Props> = props => {
     setStep(Step.chooseImage);
   }, [setStep]);
 
+  const onRetry = useCallback(() => {
+    setError(null);
+    setNonce(nonce => nonce + 1);
+  }, []);
+
+  const isRefusedOnStaxError =
+    error instanceof ImageLoadRefusedOnDevice || error instanceof ImageCommitRefusedOnDevice;
```

Comment:
hehe el famoso "The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter." -> just add an `as unknown` i guess